endpointmanager/pkg/fhirendpointquerier/populatefhirendpoints.go

```diff
@@ -48,8 +48,13 @@ func saveEndpointData(ctx context.Context, store endpointmanager.FHIREndpointSto
 	} else if err != nil {
 		return errors.Wrap(err, "getting fhir endpoint from store failed")
 	} else {
-		// @TODO (Will be addressed in LANTERN-149)
-		// Update database with this information & always overwrite the endpoint list source
+		// Always overwrite the db entry with the new data
+		existingEndpt.OrganizationName = fhirEndpoint.OrganizationName
+		existingEndpt.ListSource = fhirEndpoint.ListSource
+		err = store.UpdateFHIREndpoint(ctx, existingEndpt)
+		if err != nil {
+			return err
+		}
 		log.Infof("An endpoint for %s has already been added to the database (%s)", existingEndpt.URL, existingEndpt.OrganizationName)
```

Comment:
recommend modifying info message to say the endpoint already existed (url and source) and then note the source that's overwriting it